addon/templates/components/x-tree-node.hbs

```diff
@@ -10,11 +10,16 @@
   {{/if}}
 </span>
 {{#if checkable}}
-  <input type="checkbox" checked={{model.isChecked}} onclick={{action 'toggleCheck'}}>
+  {{input type="checkbox"
+    checked=model.isChecked
+    change=(action 'toggleCheck')
+    indeterminate=model.isIndeterminate}}
```

Comment:
Would you mind swapping `indeterminate` with `change`? I usually put the properties first and the actions last...